src/js/coblocks-lightbox.js

```diff
@@ -71,8 +71,16 @@
 		const imagePreloader = {};
 
 		Array.from(images).forEach( function( img, imgIndex ) {
+			// Enables support for lazy loading base64 URLs by using the data-src attribute.
```

Comment:
Hi, wouldn’t it be better/working for more cases to change the code so that it gets the image source on click, instead of for all images on page load? Because when the image is lazy loaded, the `src` attribute holds the correct image link. And with that change (I guess) it would also work for other lazy load solutions that use other attributes.

Reply:
Thanks for working on this ❤️

Reply:
Great feedback! Changes made. 👍